<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attendancepage</class>
 <widget class="QWidget" name="attendancepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Present Condition :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckIn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CHECK OUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckOut">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CHECK OUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPermission">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PERMISSION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
